velox/common/base/Counters.cpp

```diff
@@ -143,6 +143,10 @@ void registerVeloxMetrics() {
   DEFINE_METRIC(
       kMetricMemoryPoolReservationLeakBytes, facebook::velox::StatType::SUM);
 
+  // Tracks the count of double frees in memory allocator.
```

Comment:
nit: how about: Tracks the count of double frees in memory allocator, indicating the possibility of buffer ownership issues when a buffer is freed more than once.